common/namespace/cache.go

```diff
@@ -653,16 +629,15 @@ func (entry *CacheEntry) NotificationVersion() int64 {
 	return entry.notificationVersion
 }
 
-// IsNamespaceActive return whether the namespace is active, i.e. non global
-// namespace or global namespace which active cluster is the current cluster
-func (entry *CacheEntry) IsNamespaceActive() bool {
+// ActiveInCluster returns whether the namespace is active, i.e. non global
+// namespace or global namespace which active cluster is the provided cluster
+func (entry *CacheEntry) ActiveInCluster(clusterName string) bool {
 	if !entry.isGlobalNamespace {
 		// namespace is not a global namespace, meaning namespace is always
 		// "active" within each cluster
```

Comment:
I thought local namespace is only "active" in its home cluster.

Reply:
Honestly not sure - just keeping the existing logic